src/IROperator.cpp

```diff
@@ -2365,4 +2404,83 @@ Expr undef(Type t) {
                                 Internal::Call::PureIntrinsic);
 }
 
+namespace Internal {
```

Comment:
As a general rule, if it *can* be an external correctness test, it *should* be, to keep libHalide smaller. I think this code can be.